client/pages/routing.jsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import { Map, GoogleApiWrapper, InfoWindow } from 'google-maps-react';
+
+class Routing extends React.Component {
+  constructor(props) {
+    super(props);
+    this.onMapClicked = this.onMapClicked.bind(this);
+    this.makeRouteURL = this.makeRouteURL.bind(this);
+    this.state = {
+      vehicleLocation: {
+        lat: 40.854885,
+        lng: -88.081807
+      },
+      route: [],
+      counter: 1,
+      vehicle: {}
+    };
+
+    const vId = 2;
+    fetch(`/api/vehicleinfo/${vId}`)
+      .then(res => res.json())
+      .then(result => {
+        this.setState({ vehicle: result });
+      });
+
+  }
+
+  onMapClicked(mapProps, map, clickEvent) {
+
+    const eventRaw = JSON.stringify(clickEvent);
+    const event = JSON.parse(eventRaw);
+    const point = { counter: this.state.counter, coords: { lat: event.latLng.lat, lng: event.latLng.lng } };
+    const routeArr = this.state.route;
+    routeArr.push(point);
+    this.setState({ route: routeArr });
+    const counter = this.state.counter + 1;
+    this.setState({ counter: counter });
+    this.forceUpdate();
+
+  }
+
+  makeRouteURL() {
+    let link = 'https://www.google.com/maps/dir/';
+    for (let i = 0; i < this.state.route.length; i++) {
+      link = `${link}${this.state.route[i].coords.lat},${this.state.route[i].coords.lng}/`;
+    }
+    // eslint-disable-next-line no-console
+    console.log(link);
```

Comment:
If you don't need this anymore, please delete it.

Reply:
I needed this to show the link created